### PR TITLE
[No ticket] Footer branch pr corrections

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- fixed https://github.com/FromDoppler/doppler-webapp/pull/25 comments
- Remove unused .css files
- Add utils.js file with flattenMessages function to "es.json" and "en.json" format 
https://github.com/yahoo/react-intl/wiki/Upgrade-Guide#flatten-messages-object
- Remove css classes

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
-import { FormattedHTMLMessage } from 'react-intl';
+import { FormattedMessage } from 'react-intl';
```
Comment (lucianaDilizio) [inline src/App.js]: Is this still working good? I thought that FormattedHTMLMessage allowed use html code inside the texts. `<code>src/App.js</code>` still showing itself fine?

Comment (fcoronelmakingsense) [reply src/App.js]: No. but <code>src/App.js</code> its an example code, non final content

File: src/App.js [hunk 2]
```diff
@@ -95,18 +95,13 @@ class App extends Component {
   render() {
     return (
       <div className="App">
-        {this.state.user ? <Header className="App-header" user={this.state.user} /> : null}
-        <img src={logo} className="App-logo" alt="logo" />
+        <Header user={this.state.user} />
```
Comment (andresmoschini) [inline src/App.js]: I am not sure about this change. https://github.com/FromDoppler/doppler-webapp/pull/25/files#r255034909

Let's talk about it.

File: src/App.js (hunk 1 above)
Comment (andresmoschini) [reply src/App.js]: No entiendo, luego de este cambio funciona mal?

File: src/index.js [hunk 3]
```diff
@@ -16,7 +17,7 @@ const messages = {
 const language = navigator.language.toLowerCase().split(/[_-]+/)[0] || 'en';
 
 ReactDOM.render(
-  <IntlProvider locale={language} messages={messages[language]}>
+  <IntlProvider locale={language} messages={flattenMessages(messages[language])}>
```
Comment (andresmoschini) [inline src/index.js]: ❤️

File: src/App.js (hunk 1 above)
Comment (cbernat) [reply src/App.js]: Yes we can use `FormattedHTMLMessage` when needed only, In the case we have pure text, this `FormattedMessage` should do.

Comment (andresmoschini) [reply src/App.js]: Entendido, _FormattedMessage_ es limitado, pero creo que algún tipo de formatos permite.

Igualmente, como dice Fede, no necesitamos ese contenido.

Comment (fcoronelmakingsense) [reply src/App.js]: i removed now

**Comment by fcoronelmakingsense** [comment]
test this please

**Pull request merged**